flight/Modules/Actuator/actuator.c

```diff
@@ -195,15 +194,23 @@ static float get_curve2_source(ActuatorDesiredData *desired, SystemSettingsAirfr
 	case MIXERSETTINGS_CURVE2SOURCE_ACCESSORY0:
 	case MIXERSETTINGS_CURVE2SOURCE_ACCESSORY1:
 	case MIXERSETTINGS_CURVE2SOURCE_ACCESSORY2:
-	case MIXERSETTINGS_CURVE2SOURCE_ACCESSORY3:
-	case MIXERSETTINGS_CURVE2SOURCE_ACCESSORY4:
-	case MIXERSETTINGS_CURVE2SOURCE_ACCESSORY5:
 		(void) 0;
-		AccessoryDesiredData accessory;
 
-		if (AccessoryDesiredInstGet(source - MIXERSETTINGS_CURVE2SOURCE_ACCESSORY0,&accessory) == 0)
-			return accessory.AccessoryVal;
-		return 0;
+		int idx = source - MIXERSETTINGS_CURVE2SOURCE_ACCESSORY0;
+
+		if (idx < 0) {
```

Comment:
This implies that something is very wrong, i.e. the accessory options are out-of-order and/or there are other options between them. Maybe better with a compile-time check (`DONT_BUILD_IF`)?